kamaki/cli/cmds/cyclades.py

```diff
@@ -457,13 +459,14 @@ class server_create(_CycladesInit, OptionalOutput, _ServerWait):
     required = ('server_name', 'flavor_id', 'image_id')
 
     @errors.Cyclades.cluster_size
-    def _create_cluster(self, prefix, flavor_id, image_id, size):
+    def _create_cluster(self, prefix, flavor_id, image_id, key_name, size):
```

Comment:
I think `_create_cluster` only takes the non optional arguments as parameters.  `key_name` should be set like the `personality` or `project_id` that are optional.